packages/ra-data-simple-prisma/src/defaultHandler.ts

```diff
@@ -27,7 +27,7 @@ import { updateManyHandler } from "./updateManyHandler";
 
 export const defaultHandler = async (
   req: RaPayload,
-  prisma: PrismaClient | PrismaClientExtends,
+  prisma: PrismaClient,
```

Comment:
seems they have dropped this interface 😢 was so useful, not it's not possible to pass an extended client without casting